oidc-wp/openid-connect-generic
# [Release] Bug fixes release

### All Submissions:






### Changes proposed in this Pull Request:



- Fixes #105 : issue with post password submission results in SSO
  authentication when "auto" mode is enabled.
- Fixes #258 issues with TravisCI builds failing due to Composer updates
  occurring incorrectly during builds.
- Fixes to allow for WP version changes according to build matrix.
- Updates TravisCI build matrix.
- Fixes #260 by changing conditional check for logout redirect URL handling for Auto Login setting.
- Limits TravisCI matrix builds for pull requests & feature/fix branches.

### How to test the changes in this Pull Request:

- TravisCI builds should now pass.

#### How to test Post Passwords handling changes:
1. Create a Page/Post and publish with a password.
2. Navigate to the Page/Post without being logged into the site.
3. Enter password, you should be authenticated and redirected back to the unlocked Page/Post.

#### How to test Auto Login URL Redirect handling changes:
1. Set plugin to use "Auto Login".
2. Logout current user.
3. After confirming logout at IDP user should be redirected back to the "Site Address" by default.

### Other information:
